_posts/2020-12-31-template.markdown

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Is this file supposed to contain anything?

Reply:
No extra content. This is the default template that people can copy to create posts.